README.md

```diff
@@ -271,8 +271,8 @@ Example response body:
          "hmrcCTUTR": "1111111111",
          "ctStatus": {
             "ctutr": "1234567890",
-            "startDate": "20201001",
-            "endDate": "20193009"
+            "startDate": "2020-10-01",
+            "endDate": "2019-30- jkhu09"
```

Comment:
is this a typo?

Reply:
whoops, yeah :laughing: good spot!

Reply:
fixed [here](https://github.com/hmrc/hec/pull/63/commits/f3aa049508ee5c9782ff3e3ff1c9de29888d4451)